ppinot-model/src/main/java/es/us/isa/ppinot/evaluation/logs/XESLog.java

```diff
@@ -0,0 +1,231 @@
+package es.us.isa.ppinot.evaluation.logs;
+
+import es.us.isa.bpmn.handler.Bpmn20ModelHandler;
+import es.us.isa.bpmn.xmlClasses.bpmn20.TFlowElement;
+
+import org.eclipse.collections.impl.lazy.parallel.set.sorted.SynchronizedParallelSortedSetIterable;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamConstants;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+import javax.xml.stream.events.XMLEvent;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * MXMLLog
+ * Copyright (C) 2013 Universidad de Sevilla
+ *
+ * @author resinas
+ */
+public class XESLog extends AbstractLogProvider {
+    private static final Logger log = Logger.getLogger(XESLog.class.getName());
+
+    
+    private static final String TAG_TRACE = "trace";
+	private static final String ATTRIBUTE_ID = "concept:name";
+	private static final String TAG_EVENT = "event";
+	private static final String TAG_LOG = "string";
+	private static final String TAG_TIME = "date";
+	private static final String TAG_TIMESTAMP = "time:timestamp";
+    private static final String TAG_EVENTTYPE = "lifecycle:transition";
+    private static final String TAG_ORIGINATOR = "org:resource";
+    
+    private XMLStreamReader reader;
+
+    private ElementName elementName;
+    private Bpmn20ModelHandler bpmn20ModelHandler;
```

Comment:
This is not necessary for the XES log, so you can remove the attribute and the parameter in the constructor